tests/dialects/test_arith.py

```diff
@@ -0,0 +1,104 @@
+import pytest
+
+from xdsl.dialects.arith import (Addi, Constant, DivUI, DivSI, Subi,
+                                 FloorDivSI, CeilDivSI, CeilDivUI, RemUI,
+                                 RemSI, MinUI, MinSI, MaxUI, MaxSI, AndI, OrI,
+                                 XOrI, ShLI, ShRUI, ShRSI, Cmpi, Addf, Subf,
+                                 Mulf, Divf, Maxf, Minf)
+from xdsl.dialects.builtin import i32, f32
+
+
+class Test_integer_arith_construction:
+    a = Constant.from_int_and_width(1, i32)
+    b = Constant.from_int_and_width(1, i32)
+
+    def test_Addi(self):
+        _ = Addi.get(self.a, self.b)
```

Comment:
Should we check that both operands are correctly `self.a` and `self.b`?

Reply:
Can do